src/test/skript/tests/regressions/2538-new MooshroomData class.sk

```diff
@@ -0,0 +1,5 @@
+test "entity data mooshroom":
```

Comment:
Missed in review. Refer to issue, not PR in regtest name.

Reply:
just to verify, are you meaning the script name "2538-new MooshroomData class.sk" should refer to the issue, or the test name `test "entity data mooshroom":` or both?

Reply:
renamed!